system-variables.md

```diff
@@ -1840,6 +1840,26 @@ mysql> SELECT job_info FROM mysql.analyze_jobs ORDER BY end_time DESC LIMIT 1;
 - 默认值：`ON`
 - 该变量来用控制是否开启 GOGC Tuner。
 
+### `tidb_gogc_tuner_max_value` <span class="version-mark">从 v7.5.0 版本开始引入</span>
```

Comment:
To maintain consistency, please ensure all system variables are listed in alphabetical order. Specifically, position `tidb_gogc_tuner_max_value` and `tidb_gogc_tuner_min_value` ahead of `tidb_gogc_tuner_threshold`.